src/components/SelectionList/RadioListItem.tsx

```diff
@@ -8,21 +8,24 @@ function RadioListItem({item, showTooltip, textStyles, alternateTextStyles}: Rad
     const styles = useThemeStyles();
 
     return (
-        <View style={[styles.flex1, styles.alignItemsStart]}>
-            <TextWithTooltip
-                shouldShowTooltip={showTooltip}
-                text={item.text}
-                textStyles={textStyles}
-            />
-
-            {!!item.alternateText && (
+        <>
+            <View style={[styles.flex1, styles.alignItemsStart]}>
```

Comment:
tiny bug: the checkbox should have the hand ✋  cursor style.

https://github.com/Expensify/App/assets/29673073/e005cd04-b43e-40a9-ba63-501afa4b828a

Reply:
I believe this is a separate issue, as the bug also appears on the members page. Should we address it in the current PR or open a new bug issue for it?

Reply:
ah okay, new bug then :)